src/test/integration/boto3/tests/test_boto3.py

```diff
@@ -15,17 +15,17 @@ def assert_is_version(self, version: str):
         self.assertTrue(patch.isdigit())
 
     def assert_have_env_vars(self, root: dict):
-        collected_envs: str = root["resource"]["process.environ"]
+        collected_envs: str = root["resource"]["attributes"]["process.environ"]
```

Comment:
do we need to change anything in the BE?

Reply:
No need of BE changes, this is not a change in the OTLP we export to SaaS, it is a change in the format of the `ConsoleExporter` that makes it look more like actual OTLP.